Sources/MongoSwift/MongoCollection+Write.swift

```diff
@@ -94,6 +94,7 @@ extension MongoCollection {
     ) -> EventLoopFuture<UpdateResult?> {
         let modelOptions = ReplaceOneModelOptions(
             collation: options?.collation,
```

Comment:
some of these changes can be reverted

Reply:
removed all `comment = options?.comment ` instances